handlers/jwks/wrapper.go

```diff
@@ -0,0 +1,71 @@
+package jwks
+
+import (
+	"crypto/ecdsa"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/zalando/planb-tokeninfo/keyloader/openid/jwk"
+	"math/big"
+	"reflect"
+)
+
+type jwksWrapper struct {
+	keys map[string]interface{}
+}
+
+func (j *jwksWrapper) MarshalJSON() ([]byte, error) {
+	m := make(map[string][]map[string]string)
+
+	keys := make([]map[string]string, len(j.keys))
+	i := 0
+	for k, v := range j.keys {
+		key, ok := v.(jwk.JSONWebKey)
+		if !ok {
+			fmt.Printf("Key %q is not a JWK\n", k)
+			return nil, fmt.Errorf("Invalid JWK: %v\n", v)
+		}
+		m, err := fromJwk(key)
+		if err != nil {
+			fmt.Printf("Failed to convert the JWK %v to a map: %v\n", key, err)
+			return nil, err
+		}
+		keys[i] = m
+		i++
+	}
+
+	m["keys"] = keys
```

Comment:
this map is a bit confusing with the 'm' var in the loop, and triggers circular structure alert in the reader. can we remove the outer 'm' var, and just call below simply:

```
return json.Marshal(map[string][]map[string]string{"keys": keys})
```

?